### ISSUE TITLE
Cannot format empty file with only a comment

### ISSUE THREAD

**Issue by timotheeguerin** [microsoft/typespec#1448]
```
["ERROR" - 8:21:37 AM] Error formatting document.
["ERROR" - 8:21:37 AM] Cannot read properties of undefined (reading 'trim')
TypeError: Cannot read properties of undefined (reading 'trim')
	at Object.ensureAllCommentsPrinted (/Users/timotheeguerin/dev/azsdk/cadl-azure/common/temp/node_modules/.pnpm/prettier@2.7.1/node_modules/prettier/index.js:8109:55)
````

[Playground Link](https://cadlplayground.z22.web.core.windows.net/?c=Ly8gZm9ybWF0)

**Comment by markcowl** [comment]
est: 3

**Comment by timotheeguerin** [comment]
Fix will be release as hotfix `compiler: 0.45.2`
